packages/core/EventManager.ts

```diff
@@ -388,8 +389,7 @@ export default class EventManager {
 
         log.debug("RescheduleOrganizerChanged: Creating Event and Meeting for for new booking");
 
-        const newEvent = { ...evt, destinationCalendar: newDestinationCalendar };
-        const createdEvent = await this.create(newEvent);
```

Comment:
why do we need to pass newDestinationCalendar to reschedule function?

Reply:
Isn't this needed for when a reschedule is assigned to a different rr host?

Reply:
then we can update the evt in handleNewBooking instead of doing creating a new evt here

Reply:
I think in line https://github.com/calcom/cal.com/pull/14109/files/01d2f68cac33a3a94395ec5ae7ab826db7d44aa0#diff-f028774ebd744c3141fb60f0fb171c59a2ae4773b285b70f834114e7a3d7afa2L387 we need the destinationCalendar of the old host and in this line (392) we need the destinationCalendar of the new host, no?